server/service/client_scripts.go

```diff
@@ -7,13 +7,25 @@ import (
 	"io"
 	"net/http"
 	"strings"
+	"time"
 
 	"github.com/fleetdm/fleet/v4/server/fleet"
 )
 
+const pollWaitTime = 5 * time.Second
+
 func (c *Client) RunHostScriptSync(hostID uint, scriptContents []byte, scriptName string, teamID uint) (*fleet.HostScriptResult, error) {
```

Comment:
Shouldn't we rename this method to Async?

Reply:
I'm hesitant because we already have a `RunHostScriptAsync` method that doesn't wait for responses (basically just returns the execution ID)